limitador-server/src/main.rs

```diff
@@ -141,6 +143,21 @@ impl Limiter {
             parsed_url.password().unwrap_or_default(),
         );
 
+        let consistency: Option<Consistency> = match env::var(INFINISPAN_COUNTERS_CONSISTENCY_ENV) {
+            Ok(env_value) => match env_value.try_into() {
+                Ok(consistency) => Some(consistency),
+                Err(_) => {
+                    eprintln!("Invalid consistency mode, will apply the default");
+                    None
+                }
+            },
+            Err(_) => None,
+        };
```

Comment:
You can use a more functional/monadic style such as `let consistency = env::var(..).ok().and_then(|v| Consistency::try_from(v).ok());` or similar.

Reply:
Your solution is missing the part of printing an error. After adding that, the resulting code it's not so much better, in my opinion.